_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1375,100 +1465,110 @@ include {
-terragrunt apply -parallelism=4
+$ aws-vault add root-iam-user
+Enter Access Key Id: XXXXXXXXXXXX
+Enter Secret Key: YYYYYYYYYYYY
 ----
 
-IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=4`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
+Next, https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html[install the AWS CLI], and check that
+authentication is working:
 
-After `apply` completes, the module will output the encrypted passwords for `alice`, `bob`, and `carol`:
+[source,bash]
+----
+aws-vault exec root-iam-user -- aws sts get-caller-identity
+----
 
-[source,hcl]
+You should get JSON output with information about your IAM user:
+
+[source,json]
 ----
-user_passwords = {
-  "alice" = "wcBMA7E6Kn/t1YPfAQgAVSXlUzumcs4UyO8E5q099YnnU="
-  "bob" = "wcBMA7E6Kn/t1YPfAQgACgbdb1mYtQx7EL4hnVWtYAi="
-  "carol" = "wcBMA7E6Kn/t1YPfAQgACgbdb1mYtQx7EL4hnVWtYAi="
+{
+  "UserId": "AIDAXXXXXXXXXXXX",
+  "Account": "<ROOT_ACCOUNT_ID>",
+  "Arn": "arn:aws:iam::<ROOT_ACCOUNT_ID>:user/<YOUR_IAM_USER>"
 }
 ----
 
-Send the encrypted password to each user, along with their user name, and the IAM user sign-in URL for the account. Each
-user can then decrypt the password on their own computer (which should have their PGP key) as follows:
+You're now ready to deploy the `account-baseline` module in the root account by running `terragrunt apply`:
 
 [source,bash]
 ----
-echo "<PASSWORD>" | base64 --decode | keybase pgp decrypt
+cd infrastructure-live/root/_global/account-baseline
+aws-vault exec root-iam-user -- terragrunt apply -parallelism=2
 ----
 
-One of the other useful piece of information you'll find once `apply` completes is the `accounts` output variable,
-which will list the account IDs for each of the child accounts that were created. Take note of these account IDs, as
-you'll need them shortly!
-
-[[lock_down_root_user]]
-=== Lock down the root account root user
+IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=2`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
```

Comment:
NIT: Consider breaking out the `ulimit` section into a separate `IMPORTANT` block. It is easy to gloss that over after reading the first sentence of this block, since it's all in one paragraph.

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/1d9b5b79c84044d7f8fb6225df4379a2399bdfe8 and https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/c6a37ff8c87fd62b63173a56e3b286cff2f81b8b.